plugins/router/template/router.go

```diff
@@ -76,6 +86,8 @@ func newTemplateRouter(templates map[string]*template.Template, reloadScriptPath
 		certManager:            certManager,
 		defaultCertificate:     defaultCertificate,
 		defaultCertificatePath: "",
+		peerEndpointsKey:       peerEndpointsKey,
+		peerEndpoints:          map[string]Endpoint{},
```

Comment:
Is ```peerEndpoints``` populated somewhere else in this file? The only place I could find it was in ```DeleteEndpoints```.